Makefile

```diff
@@ -1,42 +1,50 @@
 DOCKER_REGISTRY ?= gcr.io/linkerd-io
 REPO = $(DOCKER_REGISTRY)/proxy-init
 
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: ## Show this help
+	@echo 'Info: For integration test using KinD, run: make kind-load && make integration-test'
+	@echo 'Info: For integration test using minikube, run: make integration-test'
```

Comment:
I think you can just say something like "For other environments, run `make integration-test` after having uploaded the images" given this might be run in other places like cloud environments.